README.md

```diff
@@ -19,27 +19,17 @@ npm install react-rhino
 yarn add react-rhino
 ```
 
-## Why?
-Why you should consider Rhino over other alternatives.
+## Usage
 
-### 🏋️‍♂️ Lightweight
-Only **~560** bytes when Minified & Gzipped.
-
-### 🐱‍🏍 Syntax you already know
-Offers a simple and straightforward syntax similar to the built-in hooks of React.
-
-### 👷‍♂️ Easy to Extend
-Add a new global state with just a single line of code, it's that simple.
+### Getting started
+Set up state management for your application following these three easy steps shown in the example. Or see a  [demo project](https://codesandbox.io/s/react-rhino-example-svv5b) showing `react-rhino` in use.
 
-### 📐 Easy to Use
-Learn how to use Rhino in a short amount of time.
 
-## Online Playground
-[Here](https://codesandbox.io/s/react-rhino-example-svv5b) is a demo project for you to get familiar with React Rhino.
+1. ### Create global state
```

Comment:
I see how a step by step instruction would do better, but here the hierarchy of the Heading tags are not correct .

eg: Both "Getting started" & "Create global state" are h3 tags